### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by montezume**
#### Chore

Update deps

* Closes #781 
* Closes #780 
* Closes #782 
* Closes #783 
* Closes #779

### CODE REVIEW

File: packages/mc-scripts/package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "postcss": "7.0.17",
     "postcss-color-mod-function": "3.0.3",
     "postcss-custom-media": "7.0.8",
-    "postcss-custom-properties": "8.0.10",
+    "postcss-custom-properties": "9.0.1",
```
Comment (montezume) [inline packages/mc-scripts/package.json]: requires node 8 or higher

File: packages/mc-scripts/package.json [hunk 2]
```diff
@@ -40,13 +40,13 @@
...
-    "babel-plugin-react-intl": "3.2.1",
-    "browserslist": "4.6.2",
+    "babel-plugin-react-intl": "3.3.3",
+    "browserslist": "4.6.3",
     "clean-webpack-plugin": "1.0.1",
     "core-js": "3.1.4",
     "css-loader": "2.1.1",
     "cssnano": "4.1.10",
-    "file-loader": "3.0.1",
+    "file-loader": "4.0.0",
```
Comment (montezume) [inline packages/mc-scripts/package.json]: minimum required nodejs version is 8.9.0

File: packages/mc-scripts/package.json (hunk 1 above)
Comment (montezume) [reply packages/mc-scripts/package.json]: hm, this is breaking our builds, resulting in "unknown word" errors

Comment (montezume) [reply packages/mc-scripts/package.json]: [ref](https://github.com/postcss/postcss-custom-properties/issues/186)

Comment (tdeekens) [reply packages/mc-scripts/package.json]: Wät.

File: packages/mc-scripts/package.json (hunk 2 above)
Comment (tdeekens) [reply packages/mc-scripts/package.json]: Should be fine under development I think.

**Comment by montezume** [comment]
After downgrading the postcss plugin, the packages build now and playground runs.

**Comment by tdeekens** [comment]
Some prop-types seem to blow.

**Comment by emmenko** [comment]
Sometimes I wonder why are we starting the week by getting into the dependency update mess, to then spend half a day (or more) fixing that mess 🙃

**Comment by montezume** [comment]
@tdeekens 🙏

**Pull request merged**